server/database/sequelize.ts

```diff
@@ -32,3 +34,42 @@ export const sequelize = new Sequelize(url, {
     idle: 10000,
   },
 });
+
+export const migrations = new Umzug({
+  migrations: {
+    glob: ["migrations/*.js", { cwd: path.resolve("server") }],
+    resolve: ({ name, path, context }) => {
+      // eslint-disable-next-line @typescript-eslint/no-var-requires
+      const migration = require(path as string);
+      return {
+        name,
+        up: async () => migration.up(context, Sequelize),
+        down: async () => migration.down(context, Sequelize),
+      };
+    },
+  },
+  context: sequelize.getQueryInterface(),
+  storage: new SequelizeStorage({ sequelize }),
+  logger: {
+    warn: (params) => Logger.warn("database", params),
+    error: (params) =>
+      Logger.error(
+        params.message as string,
+        (params as unknown) as MigrationError
```

Comment:
Safe to assume it as [`MigrationError`](https://github.com/sequelize/umzug#errors) :+1:

Reply:
Probably better off casting in the method params then eg `error: (params: MigrationError) =>` or similar?

Reply:
Better :+1: